docker-image/ansible/roles/traefik/tasks/main.yml

```diff
@@ -11,5 +11,7 @@
   register: traefik_config_file_status
 
 - name: "Start traefik on the cluster"
-  when: traefik_stack_file_status.changed|bool or traefik_config_file_status.changed|bool
+  when:
+    - groups['edge'][0] == inventory_hostname
```

Comment:
You may have more than one load balancer in the cluster

Reply:
The stack expands the container in "Global".
If you deploy the stack from multiple edges simultaneously, they will collide.